tasks/update-code/git.yml

```diff
@@ -1,6 +1,30 @@
 ---
+
+- name: ANSISTRANO | GIT | Ensure GIT deployment key is up to date
+  copy:
+    src: "{{ ansistrano_git_identity_key_path }}"
+    dest: ".ssh/git_identity_key"
```

Comment:
My question is about the `dest` param, not the `src`.

I think there will be a problem if two apps are deployed to the same host with different key files, and if this path is relative to the user `$HOME` directory. Each time a new release is deployed, the key file is overwritten. I think the destination path should be relative to `{{ansistrano_deploy_to}}` to avoid this.